docs/en/styling-and-theming/introduction.md

```diff
@@ -0,0 +1,13 @@
+# Introduction
+
+As a HTML-based technology, Dojo makes use of CSS for styling elements across the framework and for applications developed with it.
```

Comment:
"an" before "HTML" (ref: [APA style](https://blog.apastyle.org/apastyle/2012/04/using-a-or-an-with-acronyms-and-abbreviations.html))

Reply:
Ohhhh... I just remembered which side of the pond you're on... up to you I guess!